src/data/timer-first-interaction.js

```diff
@@ -0,0 +1,8 @@
+const listener = document.addEventListener('pulse-send-pagemonitor', evt => {
```

Comment:
This module's overall pattern seems to repeat a lot. Might be nice to abstract into a shared utility function if possible. Can't remember if these scripts are getting run through webpack, though.